### DISCUSSION TITLE
Not able to access postman

### DISCUSSION THREAD

**Discussion by unamdev0** [keyshade-xyz/keyshade#410] [Q&A] [answered]
Hello, I'm trying to setup keyshade locally to contribute with the codebase, but I'm not able to access the postman link provided [here](https://docs.keyshade.xyz/contributing-to-keyshade/running-things-locally/api-testing). This is the [link](https://www.postman.com/blue-crescent-581920/workspace/keyshade) which is provided. @rajdip-b, @kriptonian1  Can you guys please provide me with the updated link if that's available,Thanks.

**Comment by rajdip-b** [comment]
Hey, thanks for pointing this out to us! We did make some changes to our Postman team, hence some of the URLs got outdated. You can use these links for now: 
- keyshade team: https://www.postman.com/keyshade/workspace/keyshade/overview
- keyshade API: https://www.postman.com/keyshade/workspace/keyshade/api/a31bdb66-69e3-469b-afb4-f2051385e634?action=share&creator=32733901

Also, not to mention, but we would be REALLY very glad if you can open up a PR to fix the broken links. Ideally, we would want you to look in these two places for broken links:
- The README.md file
- The official docs (whose files you can find under `docs/`)

**Reply by unamdev0** [reply]
Thank you, works.

Will open a PR and fix this in `docs` , since there's no mention of postman collection in README.md, no changes are required there.
However in `CONTRIBUTING.md` there are these [two](https://github.com/keyshade-xyz/keyshade/blob/223cb39b3fca94d3550ff5db2a3e6041e8fb0e3a/CONTRIBUTING.md?plain=1#L41) [links](https://github.com/keyshade-xyz/keyshade/blob/223cb39b3fca94d3550ff5db2a3e6041e8fb0e3a/CONTRIBUTING.md?plain=1#L42) which seems to be broken. Please let me know what should be done for these two links

**Reply by rajdip-b** [reply]
Yes! My bad, they were moved to `CONTRIBUTING.md`. Please fix them aswell :D

**Reply by unamdev0** [reply]
sure, will do :⁠-⁠)

**Reply by rajdip-b** [reply, marked as answer]
@unamdev0 hey bro, I have updated the links :) you can start with other contributions if you would like to!

**Reply by unamdev0** [reply]
cool thanks, will do

**Reply by unamdev0** [reply]
Hey @rajdip-b  there seemed to be a small typo in your fix 😅 fixed that,please merge 
https://github.com/keyshade-xyz/keyshade/pull/411

**Reply by rajdip-b** [reply]
Wow haha! That was a nice catch lol.
